### PR TITLE
PR: Add setup.py

### LINKED ISSUE #21: Create a pypi package

### PR DESCRIPTION
**Pull request by dalthviz**
Fixes #21

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+"""
+
+# Third party imports
+from setuptools import find_packages, setup
+from spyder_notebook import __version__
+
+REQUIREMENTS = ['spyder', 'notebook']
+
+setup(
+        name='spyder-notebooks',
```
Comment (ccordoba12) [inline setup.py]: `spyder-notebooks` -> `spyder-notebook`

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        version=__version__,
+        keywords=["github changelog milestone"],
```
Comment (ccordoba12) [inline setup.py]: Please change this to `["spyder", "jupyter", "notebook"]`

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        license='MIT',
+        author='Spyder Development Team',
+        description='Jupyter notebook integration with Spyder',
+        long_description="This is a initial implementation of "
+                         "the notebook running inside Spyder "
+                         "as a plugin.",
+        packages=find_packages(exclude=['contrib', 'docs', 'tests*']),
+        install_requires=REQUIREMENTS,
+        classifiers=[
+                     'Development Status :: 5 - Production/Stable',
```
Comment (ccordoba12) [inline setup.py]: This should be

    'Development Status :: 4 - Production/Stable'

We're not ready for the prime time yet ;-)

File: setup.py [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+                     'Intended Audience :: Developers',
+                     'License :: OSI Approved :: MIT License',
+                     'Operating System :: OS Independent',
+                     'Programming Language :: Python 2.7',
+                     'Programming Language :: Python 3.5'
```
Comment (ccordoba12) [inline setup.py]: Please change this to

    'Programming Language :: Python 3'

**Comment by ccordoba12** [comment]
Glad to see you working on this again!! I made some simple comments, then this should be ready ;-)

File: setup.py [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder_notebook/__init__.py for details)
+
+
+"""
+Spyder-notebook
+======
+
+Jupyter notebook integration with Spyder
```
Comment (goanpeca) [inline setup.py]: `.` at the end

File: setup.py [hunk 6]
```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder_notebook/__init__.py for details)
+
+
+"""
+Spyder-notebook
+======
```
Comment (goanpeca) [inline setup.py]: complete the `=` to match the length

File: setup.py [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+===============
+
+Jupyter notebook integration with Spyder.
+"""
+
+# Third party imports
+from setuptools import find_packages, setup
+from spyder_notebook import __version__
+
+REQUIREMENTS = ['spyder', 'jupyter', 'notebook']
```
Comment (ccordoba12) [inline setup.py]: Requirements were fine. Please leave them as spyder and notebook.

File: setup.py [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+
+setup(
+        name='spyder-notebook',
+        version=__version__,
+        keywords=["github changelog milestone"],
```
Comment (ccordoba12) [inline setup.py]: It's keywords what I want you to change to spyder, jupyter, notebook

File: setup.py (hunk 7 above)
Comment (ccordoba12) [inline setup.py]: And even better: spyder>=3, notebook

File: setup.py (hunk 8 above)
Comment (ccordoba12) [inline setup.py]: Please verify if this has to be a single string or multiple ones

File: setup.py [hunk 9]
```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder_notebook/__init__.py for details)
+
+
+"""
+Spyder-notebook
```
Comment (ccordoba12) [inline setup.py]: Please change this to

    Spyder notebook plugin

**Comment by ccordoba12** [comment]
Looks good now, thanks @dalthviz!

**Pull request merged**